python/cuml/test/test_fil.py

```diff
@@ -129,6 +130,60 @@ def test_fil_classification(n_rows, n_columns, num_rounds, tmp_path):
     assert array_equal(fil_preds, xgb_preds_int)
 
 
+@pytest.mark.parametrize('n_rows', [10000])
+@pytest.mark.parametrize('n_columns', [20])
+@pytest.mark.parametrize('n_estimators', [1, 10])
+@pytest.mark.parametrize('max_depth', [2, 10, 20])
+@pytest.mark.parametrize('storage_type', ['DENSE', 'SPARSE'])
+@pytest.mark.skipif(has_treelite() is False, reason="need to install treelite")
+def test_fil_skl_classification(n_rows, n_columns, n_estimators, max_depth,
+                                storage_type):
+
+    # skip depth 20 for dense tests
+    if max_depth == 20 and storage_type == 'DENSE':
+        return
+
+    # settings
+    classification = True  # change this to false to use regression
```

Comment:
Would be nice to include a regression test too. I'm ok doing that as the next change though.

Reply:
I'll do it in the following pull request.